nautobot/extras/forms/mixins.py

```diff
@@ -837,7 +837,7 @@ def __init__(self, *args, **kwargs):
         self.order_fields(self.field_order)  # Reorder fields again
 
 
-class TagsBulkEditFormMixin(forms.Form):
+class TagsBulkEditFormMixin(BulkEditForm):
```

Comment:
Should we make similar changes to the other classes above (e.g. `RoleModelBulkEditFormMixin`, `StatusModelBulkEditFormMixin`) in the name of completeness? Or does that risk unintended side effects?

Reply:
Ah yes, both of these utilize self.model, but inherit from forms.Form which does not have the model attribute.